install.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Authors:
+#   Sreeja Kamishetty <sreeja25kamishetty@gmail.com>
+
+# Description:
+#   A post-installation bash script for engelsystem
+
+echo "Update your package manager. $ sudo apt-get update."
+apt-get update
+
+echo "Installing LAMP"
+echo "Install Apache"
+apt-get install apache2
+
+echo "Install MySQL"
+apt-get install mysql-server php5-mysql
+
+echo "Install PHP"
+apt-get install php5 libapache2-mod-php5 php5-mcrypt
+
+echo "Install php cgi"
+apt-cache search php5-cgi
```

Comment:
```
Requirements:

1.Setup LAMP

    1.1 PHP 5.4.x (cgi-fcgi)

    1.2 MySQL-Server 5.5.x pr MariaDB

    1.3 Webserver ( Apache/Nginx/lighttpd)

```
These are the requirements for the system as in the `Readme`. While I installed the system on my local server I have installed PHP-CGI and I have included in the script.